markdown/org/docs/patterns/skully/measurements/en.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Skully (the logo): Required Measurements"
+---
+
+<Note>
+Skully does not require any measurements, but it can use the head circumference
+measurement. If used, Skully will be approximately the same size as your head.
```

Comment:
```suggestion
Skully does not require any measurements, but it can use the head circumference measurement. If used, Skully will be approximately the same size as your head.
```